src/pygbag/app.py

```diff
@@ -379,6 +380,36 @@ async def main_run(app_folder, mainscript, cdn=DEFAULT_CDN):
 """
     )
 
+    config_parse = configparser.ConfigParser()
+    # config_parse["DEPENDENCIES"] = {"ignoreDirs":[], "ignoreFiles":[]}
+    if os.path.exists("pygbag.ini"):
+        config_parse.read("pygbag.ini")
+    else:
+        print("WARNING: No pygbag.ini found! See: https://pygame-web.github.io/wiki/pygbag-configuration")
+    
+    ignore_files = config_parse.get("DEPENDENCIES", "ignoreFiles")
+    ignore_dirs = config_parse.get("DEPENDENCIES","ignoreDirs")
+    def parse_ignore(ignore_raw:str):  # TODO: move this somewhere
+        if not ignore_raw:
+            return []
+        ignore_raw = ignore_raw.strip()
+        
+        for quote in ['"', "'"]:
+            if ignore_raw.find(quote) != -1:
+                print(f'ERROR: Don\'t use {quote} in pygbag config!')
+                raise SystemExit
+        
+        if ignore_raw.startswith("["):
+            if ignore_raw:
+                ignore_raw = ignore_raw.strip("[]")  # remove brackets
+                return [folder for folder in ignore_raw.split(",")]
+            else:
+                return []
+        else:  # just a single item without brackets
+            return [ignore_raw]
```

Comment:
why not just 
list(value.strip() for value in ignore_raw.splitlines())

```ini
;pygbag.ini
[DEPENDENCIES]
ignoreFiles =
    not_this_file1
    not_this_file2
    not_this_file3
```

Reply:
wouldnt ```ini
[DEPENDENCIES]
ignoreFiles = [file1, file2, file3]
```
be better? one file/folder per line would result in really long ini files

Reply:
i don't like [,] much it is not very friendly with output from "find"